cypress.json

```diff
@@ -1,5 +1,6 @@
 {
-
+  "viewportWidth": 1920,
+  "viewportHeight": 1080,
   "baseUrl": "http://localhost:3001",
```

Comment:
We probably need to implement .env file so we can just use an environmental variable as a base URL, thoughts?

Reply:
This is only for test environment for cypress. We only want to use the localhost port for this. When we setup the calls to the API we will set up an env file for these.

Reply:
Aaaah right, did not see the file is cypress.json. Thanks